curriculum/challenges/english/blocks/workshop-cafe-menu/5f33294a6af5e9188dbdb8f3.md

```diff
@@ -1,12 +1,14 @@
 ---
 id: 5f33294a6af5e9188dbdb8f3
-title: Step 5
+title: Step 1
 challengeType: 0
-dashedName: step-5
+dashedName: step-1
 ---
 
 # --description--
 
+In this workshop, you will practice the basics of CSS (Cascading Style Sheets) by building a cafe menu.
+
 It's time to add some menu content. Add a `main` element within the existing `body` element. It will eventually contain pricing information about coffee and desserts offered by the cafe.
```

Comment:
We should probably tweak the sentence it a bit. "It's time" implies some prior actions happened, but this step is now the very beginning.

```suggestion
Let's start by adding some menu content. Add a `main` element within the existing `body` element. It will eventually contain pricing information about coffee and desserts offered by the cafe.
```